.github/workflows/build-and-deploy.yml

```
@@ -61,13 +61,16 @@
           asset_content_type: application/zip
 
   create-release:
-    needs: build # This job depends on the build job
+    needs: [ build, upload-release-asset ] # This job depends on the build job
     runs-on: ubuntu-latest # Use the latest Ubuntu runner
-    if: github.event_name == 'push' && github.ref == 'refs/heads/main' # Only run this job on push events to the main branch
+    #if: github.event_name == 'push' && github.ref == 'refs/heads/main' # Only run this job on push events to the main branch
```

yamllint

[warning] missing starting space in comment

(comments)